toolkit/block-production-log/pallet/src/lib.rs

```diff
@@ -267,3 +201,30 @@ mod block_participation {
 		}
 	}
 }
+
+#[cfg(feature = "aura-compat")]
+mod aura_support {
+	use crate::{GetAuthor, GetMoment};
+	use pallet_session_validator_management::CommitteeMemberOf;
+	use sp_consensus_aura::Slot;
+
+	impl<T: crate::Config + pallet_aura::Config> GetMoment<Slot> for pallet_aura::Pallet<T> {
+		fn get_moment() -> Slot {
+			pallet_aura::CurrentSlot::<T>::get()
+		}
+	}
+
+	impl<T: crate::Config + pallet_session_validator_management::Config, Author>
+		GetAuthor<Slot, Author> for pallet_session_validator_management::Pallet<T>
+	where
+		CommitteeMemberOf<T>: Into<Author>,
+	{
+		fn get_author(slot: &Slot) -> Author {
+			pallet_session_validator_management::Pallet::<T>::get_current_authority_round_robin(
+				u64::from(*slot) as usize,
+			)
+			.expect("Block author must be known")
+			.into()
+		}
```

Comment:
having that we should be able to maybe remove this round robin assumption and aura compilation flag